changelogs/fragments/1287-new-module-vpn-tunnels.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
```

Comment:
I add this file as CI fails without it, but according to [Ansible docs](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment) it is not needed for a new module: _"You do not need a changelog fragment for PRs that: add new modules and plugins, because Ansible tooling does that automatically"_ so maybe it can be deleted?